src/disProd.js

```diff
@@ -1,7 +1,10 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable prefer-const */
 import axios from 'axios';
 import getProducts from './getProducts';
 import CommentsPopUp from './comments';
 import Config from './config';
+import productCounter from './productCounter';
```

Comment:
- [ ] productCounter' is defined but never used 
> To keep your codebase clean, It is recommended to exclude all variables that are not used within the flow of the program.

Provided that you're not using this variable in this file, kindly remove the line section and import it in the respective where it is intended to be used.